applications/stream-applications-core/pom.xml

```diff
@@ -214,17 +223,37 @@
                                         </dependency>
                                         <dependency>
                                             <groupId>io.micrometer</groupId>
-                                            <artifactId>micrometer-registry-prometheus</artifactId>
+                                            <artifactId>micrometer-registry-prometheus-simpleclient</artifactId>
+                                        </dependency>
+                                        <dependency>
+                                            <groupId>io.micrometer.prometheus</groupId>
+                                            <artifactId>prometheus-rsocket-spring</artifactId>
+                                        </dependency>
+                                        <dependency>
+                                            <groupId>io.micrometer.prometheus</groupId>
+                                            <artifactId>prometheus-rsocket-client</artifactId>
                                         </dependency>
-										<dependency>
-											<groupId>io.micrometer.prometheus</groupId>
-											<artifactId>prometheus-rsocket-spring</artifactId>
-											<version>${prometheus-rsocket.version}</version>
-										</dependency>
                                         <dependency>
                                             <groupId>io.micrometer</groupId>
                                             <artifactId>micrometer-registry-wavefront</artifactId>
                                         </dependency>
+                                        <!-- Tracing -->
+                                        <dependency>
+                                            <groupId>io.micrometer</groupId>
+                                            <artifactId>micrometer-tracing-bridge-brave</artifactId>
```

Comment:
Wouldn't be there something like OTel support request from users?

Reply:
Its possible. I would rather do that in a separate PR as this is to get feature parity w/ what was removed w/ Sleuth.